storage/connect/tabmysql.cpp

```diff
@@ -760,7 +770,8 @@ int TDBMYSQL::MakeDelete(PGLOBAL g)
   {
   char *qc, cmd[8], from[8], tab[96], end[512];
 
-  Query = (char*)PlugSubAlloc(g, NULL, strlen(Qrystr) + 64);
+  size_t Query_sz = strlen(Qrystr) + 64;
+  Query = (char*)PlugSubAlloc(g, NULL, Query_sz);
```

Comment:
allocation failure?

Reply:
The same comment as above (L745) Allocation failure handled by `PlugSubAlloc()`. We can add additional check though if you think we need